### PR TITLE
Update inf-addservice-directive.md

### PR DESCRIPTION
**Pull request by limitedge**
Added the RequiredPrivileges directive to the [service-install-section] part of the IF AddService Directive

### CODE REVIEW

File: windows-driver-docs-pr/install/inf-addservice-directive.md [hunk 1]
```diff
@@ -244,6 +245,14 @@ For information about security descriptor strings, see [Security Descriptor Defi
 
 For more information about how to specify security descriptors, see [Creating Secure Device Installations](creating-secure-device-installations.md).
 
+**RequiredPrivileges**=*privilege-name*
```
Comment (jasonknichel) [inline windows-driver-docs-pr/install/inf-addservice-directive.md:248]: Should look like Dependencies:
Dependencies=depend-on-item-name[,depend-on-item-name]...

so this should be (with appropriate formatting):
RequiredPrivileges=privilege-name[,privilege-name]...

File: windows-driver-docs-pr/install/inf-addservice-directive.md [hunk 2]
```diff
@@ -244,6 +245,14 @@ For information about security descriptor strings, see [Security Descriptor Defi
...
+> [!NOTE]
+> This value can only be used for *Win32 Services* and is only available with Windows 10 Version 2004 and above.
```
Comment (jasonknichel) [inline windows-driver-docs-pr/install/inf-addservice-directive.md:250]: This is available on Windows 7 and above, not Windows 10 Version 2004 and above.

File: windows-driver-docs-pr/install/inf-addservice-directive.md [hunk 3]
```diff
@@ -244,6 +245,14 @@ For information about security descriptor strings, see [Security Descriptor Defi
...
+
+Each *privilege-name* in the list is the name of the privilege.  See [Privilege Constants (Windows)](/windows/win32/secauthz/privilege-constants) for a list of the privilegs names. For each privilege name, only the text name is required, *For Example* The privilege name should be written as "SetAuditPrivilege" but not SE_AUDIT_NAME.
```
Comment (jasonknichel) [inline windows-driver-docs-pr/install/inf-addservice-directive.md:252]: "SeAuditPrivilege", not "SetAuditPrivilege" (you have an extra 't' in there)

**Comment by tedhudek** [comment]
Thanks for the PR @limitedge.

Looks like the fixes that @jasonknichel requested are in a new PR #3292 so closing this one

**Pull request closed without merging**